src/main/java/com/github/mojos/distribute/DeployMojo.java

```diff
@@ -0,0 +1,57 @@
+package com.github.mojos.distribute;
```

Comment:
can you add something to readme to describe this. Does it work with all distribution types, etc.

Reply:
Done.